pkg/apis/certificates/types.go

```diff
@@ -43,6 +43,10 @@ type CertificateSigningRequestSpec struct {
 	// Base64-encoded PKCS#10 CSR data
 	Request []byte `json:"request"`
 
+	// signingProfile specifies a requested signing profile for a CA to use
+	// when issuing the certificate.
+	// SigningProfile string `json:"signingProfile,omitempty"`
```

Comment:
I'm going to second @liggitt's opinion. It creates the need for further management around these profiles, while I'd prefer just having these cleaned up when the CSR is removed.